engine/src/main/resources/application.properties

```diff
@@ -221,3 +221,9 @@ quarkus.cache.caffeine.drawer-template.expire-after-write=PT5M
 # Temp - For testing purposes - Revert ASAP
 # When set to 'false', the Camel routes discovery will be disabled in the engine, preventing any Google Chat, Teams or Slack notification from being sent
 quarkus.camel.routes-discovery.enabled=false
+
+# REST client settings for the back office proxy, BOP or MBOP.
+quarkus.rest-client.mbop.url=http://localhost:8090
+quarkus.rest-client.mbop.trust-store=${clowder.private-endpoints.export-service-service.trust-store-path}
+quarkus.rest-client.mbop.trust-store-password=${clowder.private-endpoints.export-service-service.trust-store-password}
+quarkus.rest-client.mbop.trust-store-type=${clowder.private-endpoints.export-service-service.trust-store-type}
```

Comment:
Clowder refeference needs updating. I think we should refer to `bop` and not `mbop`

Reply:
I thought I had written a comment here, but it seems that I left it as a pending review. I was saying that I didn't see the MBOP service definition in Clowder, so I wasn't sure what to put here... I will double check with the access management team, just to be safe.

Reply:
After checking with them they told us that BOP/MBOP are not Clowderized applications, and therefore the parameters must be manually set in AppInterface. That is why I gave the REST Client a default value locally, but didn't set any value in Clowder, so that we are forced to set a value in stage and production.